src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemFilingContextTests.cs

```diff
@@ -50,6 +50,17 @@ public void WorkItemFilingContext_NullSarifLogRaisesArgumentNullException()
             workItemModel.Area.Should().BeNull();
         }
 
+        [Fact]
+        public void WorkItemFilingContext_PopulateDescription()
+        {
+            var context = new SarifWorkItemContext();
+            SarifLog sarifLog = TestData.SarifLogs.OneIdThreeLocations;
+
+            var workItemModel = new SarifWorkItemModel(sarifLog, context);
```

Comment:
>SarifWorkItemModel [](start = 36, length = 18)

Why is this test in the SarifWorkItemFilingContext test file? The context doesn't currently influence the description creation (though I've suggested otherwise in another PR comment. :) You should move this to WorkItemModel tests, I think, or test at a lower level in WorkItemExtesionsTests.